generators/entity-server/templates/src/main/resources/config/liquibase/changelog/added_entity.xml.ejs

```diff
@@ -82,9 +82,17 @@
             } else if (fieldType === 'LocalDate') {
                 columnType="date";
             } else if (fieldType === 'Instant') {
-                columnType="timestamp";
+                if (prodDatabaseType === 'mysql') {
```

Comment:
We can always use `datetime` in place of `timestamp` as Liquibase resolves it to the same `timestamp` where `datetime` is not supported  https://stackoverflow.com/a/28626825/3737815